components/Feedback.js

```diff
@@ -0,0 +1,110 @@
+import { useState } from "react";
+import { useRouter } from "next/router";
+
+export default function Feedback(className = '') {
+    const [submitted, setSubmitted] = useState(false);
+    const [error, setError] = useState(false);
+    const [feedback, setFeedback] = useState('');
+    const { asPath, pathname } = useRouter();
+    
+    const date_stamp = new Date()
+    const time_stamp = `${date_stamp.getDate().toString()}/${(date_stamp.getMonth()+1).toString()}/${date_stamp.getFullYear().toString()} ${date_stamp.getHours().toString()}:${date_stamp.getMinutes().toString()}:${date_stamp.getSeconds().toString()}`;
```

Comment:
I think we can use `toUTCString()` or any other similar function that we believe is enough. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date

```suggestion
    const time_stamp = date_stamp. toUTCString();
```